generators/app/templates/src/StarterKit/Enrichers/TypeEnricher.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Data;
```

Comment:
Het zou properder zijn als dit mee onder een algemene folder "logging" of iets dergelijks komt te staan onder de Framework-folder. De folder met het bestand RequestResponseLoggingMiddleware voor de Middleware zou daar dan ook onder kunnen staan. Dan staat dit functioneel tesamen en is het duidelijk waarvoor dit allemaal dient.